### ISSUE TITLE
Cannot use entity in PermissionsBuilder in 2.0.0-alpha

### ISSUE THREAD

**Issue by jabelone** [Camberi/firecms#403]
Hey there, I'm trying to update our application to 2.0.0-alpha as there are a couple features critical to our use case.

In several of our collections, we use `{ entity }` in the permissions builder to check if a user should have that permission. For example, we might check something like this:

```
buildCollection({
            ...
            permissions: ({ entity }) => {
                return {
                    edit: entity?.values?.someProperty === true,
                    create: isAdmin,
                    delete: false,
                };
            },
        }),
```

This works in our current app. However, in 2.0.0-alpha it appears the types have changed and the entity property was removed (I think in [this commit here](https://github.com/Camberi/firecms/commit/be636af217d32c57e6cfe409e90cd048e9876c0a#diff-00ebc8c7d4d0184702f978aa9143ec839644829893feb8e621252552b2c2f4b9)). I'm following the example [here](https://firecms.co/docs/collections/permissions), which still shows `{entity}` being used.

Is this an error with the types, or the documentation? And if this is no longer supported, is there a better way of setting permissions based on values in each document/entity?

Thank-you!

**Comment by fgatti675** [comment]
I think this was lost during the migration phase.
Just added it back here :)
https://github.com/Camberi/firecms/commit/368aa6f5a68482c5603ca0d6123b263e90c1ffe4
It has been included in version `2.0.0-alpha.39`

**Comment by jabelone** [comment]
Thanks so much, I'll check it out :)
